Mad.Api/Mad.Api/Controllers/WeatherForecastController.cs

```diff
@@ -48,5 +48,20 @@ public IActionResult DeleteCurrentWeatherTest()
             return Ok();
         }
 
+        [HttpPost("post-current")]
+
+        public string NotificationToCivilians()
+        {
+            var NowWeather = new WeatherForecast();
+            NowWeather.TemperatureC = Random.Shared.Next(-45, 55);
+            string alert = $"Everything is nice! Enjoy your life! The Temperature is {NowWeather.TemperatureC}";
```

Comment:
```suggestion
            var alert = $"Everything is nice! Enjoy your life! The Temperature is {NowWeather.TemperatureC}";
```
Here compiler can know what this variable have a string type, and so you can use 'var'